src/maybe.js.flow

```diff
@@ -2,7 +2,7 @@
 
 export type Maybe<A> = Just<A> | Nothing
 
-type AnyVal = number | boolean | string | Object
+type AnyVal = number | boolean | string | Object | Array<*>
```

Comment:
make it like https://github.com/facebook/react-native/blob/9ee815f6b52e0c2417c04e5a05e1e31df26daed2/Libraries/StyleSheet/StyleSheetTypes.js#L14 where it's `Array<?AnyVal>`